plugins/inventory/insights.py

```diff
@@ -35,15 +36,15 @@
         type: str
       staleness:
         description: Choose what hosts to return, based on staleness
-        default: [ 'fresh', 'stale', 'unknown' ]
+        choices: [ 'fresh', 'stale', 'unknown' ]
```

Comment:
hm I don't think this is correct: `staleness` is actually a list, and the mentioned string is the default (i.e. 3 elements)